### PR TITLE
Add pad_size arg to OSCDDataModule

### PR DESCRIPTION
**Pull request by isaaccorley**
This PR adds:
- `pad_size` argument to the `OSCDDataModule`. Previously the pad size for the `PadTo` transform used in the val/test steps was hardcoded to `(1280, 1280)`.
- Updated default OSCD configs to use `(64, 64)` pad size which is the smallest I could get a U-Net to take due to downsampling. This should reduce the memory consumption used when testing the trainers/datamodules.

### CODE REVIEW

File: conf/task_defaults/oscd_all.yaml [hunk 1]
```diff
@@ -19,3 +19,6 @@ experiment:
     val_split_pct: 0.5
     bands: "all"
     num_patches_per_tile: 1
+    pad_size:
+      - 64
+      - 64
```
Comment (adamjstewart) [inline conf/task_defaults/oscd_all.yaml:24]: Can you add an EOL char?

File: conf/task_defaults/oscd_all.yaml [hunk 2]
```diff
@@ -18,4 +18,8 @@ experiment:
     num_workers: 0
     val_split_pct: 0.5
     bands: "all"
+    pad_size:
+      - 64
+      - 64
     num_patches_per_tile: 1
+
```
Comment (adamjstewart) [inline conf/task_defaults/oscd_all.yaml:25]: Can you remove the empty line?

Comment (calebrob6) [reply conf/task_defaults/oscd_all.yaml:25]: didn't you just ask to add an empty line?

Comment (adamjstewart) [reply conf/task_defaults/oscd_all.yaml:25]: EOL char and newline char are two different things.

Comment (calebrob6) [reply conf/task_defaults/oscd_all.yaml:25]: Which chars are those? I'm only familiar with "\n" or "\r\n". (It looks like this file ends with two newlines though)

Comment (adamjstewart) [reply conf/task_defaults/oscd_all.yaml:25]: Okay, I guess you're right, previously there was no "\n", now there's "\n\n", the goal is "\n". My text editor adds this automatically. I think most editors can be configured to do this.

Comment (calebrob6) [reply conf/task_defaults/oscd_all.yaml]: kk np, just making sure I understand (char codes are wayyyy in the past for me)

**Review by calebrob6 — APPROVED** [review]

File: conf/task_defaults/oscd_rgb.yaml [hunk 3]
```diff
@@ -18,4 +18,8 @@ experiment:
     num_workers: 0
     val_split_pct: 0.5
     bands: "rgb"
+    pad_size:
+      - 64
+      - 64
     num_patches_per_tile: 1
+
```
Comment (adamjstewart) [inline conf/task_defaults/oscd_rgb.yaml:25]: This file too

**Review by adamjstewart — APPROVED** [review]

**Comment by calebrob6** [comment]
Merging this because mypy issues are unrelated.

**Pull request merged**